src/main/java/Blu/command/EventCommand.java

```diff
@@ -0,0 +1,50 @@
+package blu.command;
+
+import java.time.LocalDateTime;
+
+import blu.exception.StorageException;
+import blu.storage.Storage;
+import blu.task.Event;
+import blu.task.TaskList;
+import blu.ui.UI;
+
+/**
+ * Represents a command to add a new Event task with a specified description and event date/time.
+ */
+public class EventCommand extends Command {
+    private final String description;
```

Comment:
I like how description, fromDateTime, and toDateTime convery their respective meanings and are single-valued attribute variables.